k8s/argocd-apps/litterrally-dev.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: litterrally-dev
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/RGPosadas/LitterRally.git
+    targetRevision: setup-ci-cd
```

This and a similar line from [k8s/argocd-apps/litterrally-prod.yaml](https://github.com/RGPosadas/LitterRally/pull/2/files#diff-d6daeb7faad82bb47441f74acacc6de4R10) will need to be updated before this is merged.

```suggestion
    targetRevision: master
```

---

Thanks for the reminder!